.circleci/config.yml

```diff
@@ -127,6 +127,7 @@ jobs:
     steps:
       - checkout
       - restore_dependency_cache
+      - run: npx browser-driver-manager@1.0.4 install chrome chromedriver
```

Comment:
Since you're doing this for every package, probably best to install it as a dependency at the root so you don't have to waste time reinstalling it for every job.

Reply:
Chrome version is at V96 when I ran it on SSH, we would need to store Chrome in a cache or find a docker image that updates the chrome versions?